e2e/app/eoa/static.go

```diff
@@ -1,285 +1,45 @@
-			Role:          RoleCreate3Deployer,
-			Address:       mainnetCreate3Deployer,
-			Chains:        chainSelectorAll,
-			MinBalance:    minBalanceSmall,
-			TargetBalance: targetBalanceSmall,
-		},
-		{
-			Type:          TypeRemote,
-			Role:          RoleDeployer,
-			Address:       mainnetDeployer,
-			Chains:        chainSelectorAll,
-			MinBalance:    minBalanceMedium,
-			TargetBalance: targetBalanceMedium,
-		},
-		{
-			Type:          TypeRemote,
-			Role:          RoleProxyAdminOwner,
-			Address:       mainnetAdmin,
-			Chains:        chainSelectorAll,
-			MinBalance:    minBalanceMedium,
-			TargetBalance: targetBalanceMedium,
-		},
-		{
-			Type:          TypeRemote,
-			Role:          RolePortalAdmin,
-			Address:       mainnetAdmin,
-			Chains:        chainSelectorAll,
-			MinBalance:    minBalanceMedium,
-			TargetBalance: targetBalanceMedium,
-		},
-		{
-			Type:          TypeRemote,
-			Role:          RoleAVSAdmin,
-			Address:       mainnetAdmin,
-			Chains:        chainSelectorAll,
-			MinBalance:    minBalanceMedium,
-			TargetBalance: targetBalanceMedium,
-		},
-		{
-			Type:          TypeSecret,
-			Role:          RoleRelayer,
-			Address:       mainnetRelayer,
-			Chains:        chainSelectorAll,
-			MinBalance:    minBalanceMedium,
-			TargetBalance: targetBalanceMedium,
-		},
-		{
-			Type:          TypeSecret,
-			Role:          RoleMonitor,
-			Address:       mainnetMonitor,
-			Chains:        chainSelectorAll,
-			MinBalance:    minBalanceSmall,
-			TargetBalance: targetBalanceSmall,
-		},
-	},
+	netconf.Devnet: flatten(
+		wellKnown(anvil.DevPrivateKey0(), RoleCreate3Deployer, RoleDeployer, RoleProxyAdminOwner, RolePortalAdmin, RoleAVSAdmin),
+		wellKnown(anvil.DevPrivateKey5(), RoleRelayer),
+		wellKnown(anvil.DevPrivateKey6(), RoleMonitor),
+		fbDevAcc(),
+	),
+	netconf.Staging: flatten(
+		remote("0x4891925c4f13A34FC26453FD168Db80aF3273014", RoleProxyAdminOwner, RolePortalAdmin, RoleAVSAdmin),
+		remote("0xC8103859Ac7CB547d70307EdeF1A2319FC305fdC", RoleCreate3Deployer),
+		remote("0x274c4B3e5d27A65196d63964532366872F81D261", RoleDeployer),
+		secret("0xfE921e06Ed0a22c035b4aCFF0A5D3a434A330c96", RoleRelayer),
+		secret("0x0De553555Fa19d787Af4273B18bDB77282D618c4", RoleMonitor),
+		fbDevAcc(),
+	),
+	netconf.Testnet: flatten(
+		remote("0xEAD625eB2011394cdD739E91Bf9D51A7169C22F5", RoleProxyAdminOwner, RolePortalAdmin, RoleAVSAdmin),
+		remote("0xeC5134556da0797A5C5cD51DD622b689Cac97Fe9", RoleCreate3Deployer),
+		remote("0x0CdCc644158b7D03f40197f55454dc7a11Bd92c1", RoleDeployer),
+		secret("0x01654f55E4F5E2f2ff8080702676F1984CBf7d8a", RoleRelayer),
+		secret("0x12Dc870b3F5b7f810c3d1e489e32a64d4E25AaCA", RoleMonitor),
+		fbDevAcc(),
+	),
+	netconf.Mainnet: flatten(
+		dummy(RoleProxyAdminOwner, RolePortalAdmin, RoleAVSAdmin, RoleCreate3Deployer, RoleDeployer),
+		secret("0x07804D7B8be635c0C68Cdf3E946114221B12f4F7", RoleRelayer),
+		secret("0x07082fcbFA5F5AC9FBc03A48B7f6391441DB8332", RoleMonitor),
+		fbDevAcc(),
```

Comment:
don't think we should include this here as it's part of our fireblocks testnet workspace, not the mainnet one

Reply:
cool, replace with `dummy` for now.